src/esm_tests/cli.py

```diff
@@ -71,13 +71,6 @@ def main():
         help="Delete previous tests",
         action="store_true",
     )
-    # parser.add_argument(
```

Comment:
Thank you!! We should always opt rather for erasing code than commenting it out, in my view. Code can be retrieved from the git history if you really need.